INSTALL.ide

```diff
@@ -39,7 +39,7 @@ COMPILATION REQUIREMENTS
   install GTK+ 2.x, should you need to force it for one reason
   or another.)
 - The OCaml bindings for GTK+ 2.x, lablgtk2 with support for gtksourceview2.
-  You need at least version 2.16.
+  You need at least version 2.18.4.
```

Comment:
Please, revert back to 2.18.4, as this is the lablgtk version available in Debian stable as of today.

Reply:
OK, thanks for this valuable info. What do you think of the version test. Should it be based on a heuristic (availability of a given function)? Any other idea?

Reply:
That's a problem I don't know how to solve, maybe emit a warning if we find 2.18.0?

Reply:
So 2.18.0 = OK + warning, >= 2.18.3 OK, everything else, fail.

Reply:
Sounds good.

Reply:
Fixed.

Reply:
Thanks Théo !!